.github/workflows/_reusable_surge-deploy-preview.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         id: surge-preview-tools
         with:
           surge-token: ${{ secrets.SURGE_TOKEN_DOC }}
+          github-token: ${{ secrets.BONITA_CI_PAT }} # Avoid rate limiting produced with default GITHUB_TOKEN
```

Comment:
**thought:** The afc163/surge-preview action also do some GH API calls (it does the exact same call as the surge-preview-tools action)
So, it should probably use the PAT.
Notice that this will change the author of PR comment created/updated by the action from github-actions bot to bonita-ci bot. This has no impact for contributors.

Reply:
It seems better with PAT after some tests on bonita-doc pr